### DISCUSSION TITLE
Creat more Collection

### DISCUSSION THREAD

**Discussion by lumivn** [outline/outline#2903] [Ideas]
Getoutline is great for working but also for personal.
Now we just have one collection.
So i thinks better can creat more kind of collection like WORKING/ PERSONAL/ FREELANCER ...

**Comment by tommoor** [comment]
What is stopping you creating more collections? 😄

**Reply by lumivn** [reply]
Maybe you miss info, 
I need 2 and more independent Collection 
<img width="874" alt="Screenshot_1" src="https://user-images.githubusercontent.com/2639582/148478214-6b83ef5e-5110-4e1a-a5ad-e36e92b87222.png">

**Reply by MajinV** [reply]
Basically allowing users to maintain their own  personal and private wiki - assume you'd expect a user to be able to promote/move a personal page into the organisation collections?

**Reply by tommoor** [reply]
We should build something like this. There was an exploratory branch that introduced a area for personal notes, I think the question is whether that should be a hardcoded thing or we should just have the generic concept of sections.

It's probably worth having personal notes as a distinguished and different area.

**Comment by bmm-alc** [comment]
Personal notes is something really useful I think.
I like to have a place to keep tips and often used commands, or even temporary things in a page instead of keeping that in different text files on my disk. The question is also, should the personal text be public by default or not.

**Reply by tommoor** [reply]
There is nothing to stop you creating a collection for personal notes that's private, then star the collection so it's at the top 👍

**Reply by bmm-alc** [reply]
> There is nothing to stop you creating a collection for personal notes that's private, then star the collection so it's at the top +1

sure, but in a widespread installation, as an admin you want that to be created / available for each new user (perhaps I could create that using the API but this is not was I would expect from a software)

-- 
best

**Reply by tommoor** [reply]
Totally agreed, and I still think it would be a nice default in the app – just wanted to be clear that it's still possible with a few clicks 😄

**Comment by emtlovell** [comment]
I would find this helpful, so I could divvy out notes to different teams. i.e.
Project Team
HelpDesk Team
Engineering team could have their own individual  collections.
